### ISSUE TITLE
Manifested Ghosts can be made permanent

### ISSUE THREAD

**Issue by Crazylemon64** [ParadiseSS13/Paradise#6854]
As per title.

**Comment by lolexploit** [comment]
What happened: I made a ton of pylons with a shitload of plasteel, manifested ghosts, the pylons kept me alive to support several ghosts forever.

""""""exploit"""""

**Comment by DarkPyrolord** [comment]
That isn't the issue, there is a way to keep the ghosts manifested, no runes, healing or damage taking required, think you might be getting situations mixed up here @lolexploit

**Comment by lolexploit** [comment]
@DarkPyrolord
Could be, I hope the way I mentioned is legit. :^)

**Comment by mrturkeytoe** [comment]
From what I understand this massive salty controversy about the cult manifested ghosts is because the persom that summoned them was smart and the people fighting them were dumb and targetted the ghosts instead of them (Even though they weren't on the same z-level and maybe manifested ghosts should dissipate if the aren't on the same z-level of their summoner)

**Comment by mrturkeytoe** [comment]
Specifically smart by anticipating a hard fight and making a shitload of pylons

**Comment by Anticept** [comment]
Diminishing returns on pylons seems like a good way to go to solve this.

**Comment by Aurorablade** [comment]
This was not intended behavior and manifested ghosts are not dummy's anymore and can be damaged. I am wondering how they got enough pylons to heal themselves like that

**Comment by taukausanake** [comment]
Is it really that bad if they manage to get that many to work? A large investment should have a large payout but still it might be broken if too easy to pull off

**Comment by mrturkeytoe** [comment]
^ My only problem is they built it on the derelict station, so they basically couldn't be beaten.

**Comment by Aurorablade** [comment]
It's not what I had in mind. And the ghosts still should be able to die. If they were for some reason not dieing away from pylons we have an issue

**Comment by Anticept** [comment]
@taukausanake it's bad design to do this regardless of investment without requiring risk. While shadowlings create a lot of salt, it's relatively much better than what happened with this cult. It basically bypasses the non-cult players, yet completely stomps them in the end. That's a much worse salt.

**Comment by mrturkeytoe** [comment]
Maybe make it like, you need to invest soul shards or something to be able to get enough pylons to make the ghosts immortal so there's a chance of getting caught? Or make it so manifested ghosts can't leave the z-level they were summoned on so people have a chance to fight back by killing the summoner?

**Comment by lolexploit** [comment]
Z-level lock seems like a nice solution to no counterplay, since pylons make a giant culted area from tile conversion, it's pretty obvious where they are once are on the same z level. Then again off-station cult bases is pretty much mandatory during high server pops and has become a staple of cult rounds.

Don't forget you're sacrificing your ability to move from a single spot for the entire duration you want the ghosts kept alive. If pylons can't keep you alive to sustain the ghosts, then the problem is that you usually get absolutely clueless ghosts that have no idea how cult functions, and by the time you explain them what you need them to do you'll be near crit from damage. Sometimes they don't even realize they have been manifested and you're stuck with an afk ghost that keeps draining your life. What players do is they just use manifest ghost as a +1 for the conversion rune since anything else is just not worth the hassle if you don't have a reliable way to heal. Even then you can get griffin ghosts that just decide to fuck off the second they are manifested, forcing you to re-manifest.

You need 3 plasteel per pylon, and a few pylons to stay alive per ghost. Each ghost needs items for which you need other cult production facilities. The setup took me well over 50 plasteel and altogether quite a while to establish. Then you have to plead to ghosts to actually come to you which again wastes a lot of time, you have to equip every single one of them or else they are just naked transparent humans. I focused just on this strategy since the start of the round and the first ghosts were out and armed around 1 hour into the round.

I think there's plenty of risk and effort involved in making use of ghosts as it is.

tl;dr: lock ghosts to summoner z-level

**Comment by mrturkeytoe** [comment]
Maybe also possibly make it so if they go onto a different z-level they do double damage to the summoner instead?

**Comment by fludd12** [comment]
Clever strategies should have counterplay, not be removed. I support the double-damage idea, but pylons and summoning should otherwise not be changed.

**Comment by Crazylemon64** [comment]
This ain't "lol pylons" for those curious
